sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -308,6 +308,8 @@ abstract class RapidsShuffleThreadedWriterBase[K, V](
   private def write(records: TimeTrackingIterator): Unit = {
     withResource(new NvtxRange("ThreadedWriter.write", NvtxColor.RED)) { _ =>
       withResource(new NvtxRange("compute", NvtxColor.GREEN)) { _ =>
+        // Timestamp when the main processing begins
+        val processingStart: Long = System.nanoTime()
```

Comment:
Since the `processingTime` will encompass the `openTimeNs` as well after this change, we will double count the open time in [here](https://github.com/NVIDIA/spark-rapids/blob/5ab1cb66144280fb198656db8b4ce31c061b672e/sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala#L451). Should we fix this line as well?

Reply:
Yeah, will remove the extra openTimeNs in the line you pointed at. Thanks again